client/src/pages/EditOrganisationAccount.js

```diff
@@ -320,6 +367,17 @@ function EditOrganisationAccount(props) {
           <CustomEditAccountHeader className="h4">
             Edit Organisation Profile
           </CustomEditAccountHeader>
+          {isEmailUpdateSuccess ? (
+            <>
+              <SuccessAlert message={"Email successfully updated"} />
+            </>
+          ) : isEmailUpdateError ? (
+            <>
+              <ErrorAlert message={orgProfileState.error} />
+            </>
```

Comment:
I think you should probably add the Fragment tag. I think it's better.

Reply:
No need though since we are mostly using the short version (which is these empty tags)